src/main/ruby/core/string.rb

```diff
@@ -276,6 +276,8 @@ def tr(source, replacement)
     str.tr!(source, replacement) || str
   end
 
+  Truffle::Graal.always_split(instance_method(:tr))
```

Comment:
Since this is a modifier like `private`, I add it without extra space after the method `end`.